src/Bundle/Command/StubCommand.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+declare(strict_types=1);
```

Comment:
Can you remove this from all files. I'd like to be consistent with the rest of the project.